test/System/Element/GetHostElements_System_Test.dyn

```diff
@@ -0,0 +1,484 @@
+{
```

Comment:
screenshot in PR please 😎

Reply:
@radumg added screenshots of dyn and rvt file